src/main/java/com/litografiaartesplanchas/inventoryservice/controller/MaterialController.java

```diff
@@ -0,0 +1,30 @@
+package com.litografiaartesplanchas.inventoryservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.litografiaartesplanchas.inventoryservice.model.Material;
+import com.litografiaartesplanchas.inventoryservice.service.MaterialService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+
+@RestController
+@RequestMapping("/api/inventory")
+public class MaterialController {
+    @Autowired
+    private MaterialService materialService;
+
+    @PostMapping
```

Comment:
The path it's like `/api/v1/{service}/save`. The anotation @PostMapping needs the param **value=save**